plugins/dell-dock/fu-dell-dock-plugin.c

```diff
@@ -18,6 +18,14 @@
 #include "fu-dell-dock-common.h"
 #include "fu-dell-dock-plugin.h"
 
+DOCK_HWDEV_INFO dock_hwdev_info_hub[] = {
+    {DOCK_BASE_TYPE_SALOMON, 0x413c, 0xb06e, "USB\\VID_413C&PID_B06E&hub"},
+    {DOCK_BASE_TYPE_SALOMON, 0x413c, 0xb06f, "USB\\VID_413C&PID_B06F&hub"},
+    {DOCK_BASE_TYPE_ATOMIC, 0x413c, 0xb06e, "USB\\VID_413C&PID_B06E&atomic_hub"},
+    {DOCK_BASE_TYPE_ATOMIC, 0x413c, 0xb06f, "USB\\VID_413C&PID_B06F&atomic_hub"},
```

Comment:
you have `#define`s in the common file, use those for the second and third members

Reply:
fixed in 271ccab.